apps/api-gateway/src/services/notification_service.py

```
@@ -161,6 +164,21 @@ async def get_user_notifications(
             if is_read is not None:
                 stmt = stmt.where(Notification.is_read == is_read)
 
+            # 类型过滤
+            if type_filter is not None:
+                stmt = stmt.where(Notification.type == type_filter)
+
+            # 优先级过滤
+            if priority_filter is not None:
+                stmt = stmt.where(Notification.priority == priority_filter)
+
+            # 关键词搜索（标题或内容）
```

Ruff (0.15.2)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)